docs/UserGuide.md

```diff
@@ -555,16 +551,20 @@ Examples:
 TAB will automatically update your lesson's outstanding fees once the lesson has ended using Fees Calculator feature. 
 The Fees Calculator will account for cancelled dates and ensure that lesson fees on these dates will not be added.
 
-However, the Fees Calculator will not account for any changes to lessons that have passed. Such cases include:
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
Would it be too long for a caution?

Reply:
I feel that the whole part about behaviour is quite important though